libs/infra-next-server/src/lib/bootstrap.ts

```diff
@@ -1,3 +1,4 @@
+import '@island.is/infra-tracing'
```

Comment:
Is this supposed to be the nest-server?

Reply:
(since you added this to the next-server)

Reply:
🤦🏻
It is supposed to be in both.

Reply:
Fixed.